.github/workflows/main.yml

```diff
@@ -0,0 +1,77 @@
+name: Generate new docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'ember or ember-data'     
+        required: true
+      version:
+        description: 'The version to build, i.e. 3.20.0'     
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Clone the project repo
+        run: |
+          set -euo pipefail
+          if [$INPUT_PROJECT -eq 'ember']
+          then
+            git clone https://github.com/emberjs/ember.js
+            cd ember.js
+            git checkout "v$INPUT_VERSION"
+          elif [$INPUT_PROJECT -eq 'ember-data']
+          then
+            git clone https://github.com/emberjs/data
+            cd data
+            git checkout "v$INPUT_VERSION"
+          else
+            echo "Please specify either ember or ember-data for the project."
+          fi
+
+      - name: Clone ember-api-docs-data
+        run: git clone https://github.com/ember-learn/ember-api-docs-data
+      
+      - name: yarn install
+        run: yarn install
+
+      - name: Generate s3-docs
+        working-directory: ember-jsonapi-docs
+        run: yarn gen --project $INPUT_PROJECT --version $INPUT_VERSION
+
+      - name: Generate json-docs
+        working-directory: ember-jsonapi-docs
+        run: yarn start --project $INPUT_PROJECT --version $INPUT_VERSION
+  
+  pull-request:
```

Comment:
We cannot split this into another job without uploading the result of the previous one as an artifact and downloading it here, because each job runs in a fresh container and doesn't have access to the things you did in the previous one. Using the upload/download artifacts action is fine, especially if it's useful to retain the output of that previous job temporarily for manual inspection and debugging; but if it's otherwise not needed, it's probably easiest to just continue running this in the same job.